command/server_members.go

```diff
@@ -103,6 +115,17 @@ func (c *ServerMembersCommand) Run(args []string) int {
 	// Sort the members
 	sort.Sort(api.AgentMembersNameSort(srvMembers.Members))
 
+	if json || len(tmpl) > 0 {
+		out, err := Format(json, tmpl, srvMembers.Members)
```

Comment:
Just one minor comment for me. Do we plan to expose `Leader` information when using `-json` flag ?